### DISCUSSION TITLE
Evangelizing OpenFrames

### DISCUSSION THREAD

**Discussion by bmann** [open-frames/standard#6] [General]
Hi all! Just had a great call with @davidfurlong. Thanks for turning on discussions!

I'm very interested in seeing OpenFrames go everywhere, so I've been thinking about what the path is to get there, as well as telling a lot of people about frames to get them thinking about it.

I'm a fan of Bluesky's ATProtocol, so going to take a look at which third-party clients we could prototype support into. That also means looking at some info around third party client adoption. My intuition says that the early adopters who want to try this stuff are also the people who try third party clients. Ultimately, people have to use it and value out of it, and I think that comes from making it easy to hack/experiment/build/play at the frames layer for unique experiences per protocol.

I also think ActivityPub will continue to be relevant, in part because of Meta's Threads compatibility, so hacking OpenFrames support into the Phanpy web client could be a near term thing https://github.com/cheeaun/phanpy

I've just been riffing on a bunch of different "OpenFrames for X" trying to think about both go to market and different experience. e.g. OpenFrames for SMS, OpenFrames for Obsidian, Substack or Medium support OF by default with "subscribe" built in, following some paths around OEmbed and so on.

I'll start some other threads around those things as they move forward.

**Comment by davidfurlong** [comment]
Adding ATProtocol support to Open Frames is interesting! Thanks for the conversation @bmann :)
